### PR TITLE
Add some test cases

### PR DESCRIPTION
**Pull request by sonallux**
This PR adds some test cases. The tests are executed against a real postgres database running in a docker container. The tests are starting the database container themself.

I have also done the following minor changes:
- Fix some console log statements
- Add a `close` method to `PostgresRepository`

### CODE REVIEW

File: test/query.test.js [hunk 1]
```diff
@@ -0,0 +1,72 @@
...
+
+  afterEach(async () => {
+    await postgresRepository.close()
+    await exec(`docker stop ${CONTAINER_NAME}`)
+    await exec(`docker rm ${CONTAINER_NAME}`)
+  })
+
+  test('simple query', async () => {
+    await startDatabase()
+    await new Promise(resolve => setTimeout(resolve, 5000))
```
Comment (georg-schwarz) [inline test/query.test.js]: We should use some variable here instead of magic number.
Should we also use the sleep function from the basics package?

Comment (sonallux) [reply test/query.test.js]: Yes, using the `sleep` function is a nice idea. But I would not use variables for these magic numbers as they are highly dependent on the individual test case.

Comment (georg-schwarz) [reply test/query.test.js]: Well, then at least name it somehow as variable so that we know what the timeout intends to do :) like `databseStartupTime`

Comment (sonallux) [reply test/query.test.js]: I have put all the magic numbers in variables.

**Review by georg-schwarz — APPROVED** [review]
Nice!

**Pull request merged**